maintainer/channels.py

```diff
@@ -0,0 +1,145 @@
+import requests
+import configuration
+
+
+class ChannelsMaintanance(object):
+    '''
+    Class is using Teamspeak API in order to maintain channels
+    which are not used.
+    '''
+    def __init__(self, qtime=2):
+        '''
+        Input:
+            qtime = time in seconds after which we are taking steps:
+                #todo fix below
+                first - quarantine status in channel_topic
+                second - after same amount of time if channel_topic is quarantined
+                we are deleting this channel
+        Init doesn't return anything
+        '''
+        self.qtime = qtime
+        self.connection = 'http://193.70.3.178:9652/'
+        self.password = configuration.auth_pwd
+        self.username = configuration.auth_user
+
+    def _requests_get(self, endpoint):
+        return requests.get('{}{}'.format(
+            self.connection, endpoint), auth=(self.username, self.password)).json()
+
+    def _requests_delete(self, endpoint):
+        requests.delete('{}{}'.format(
+                self.connection, endpoint), auth=(self.username, self.password))
+
+    def _requests_post(self, endpoint, payload):
+        requests.post('{}{}'.format(self.connection, endpoint), auth=(
+            self.username, self.password), data=payload)
+
+    def channels_basic_info(self):
+        '''
+        Get basic information about every channel on server.
+        In details it tells us names of channels, how many users are on specific
```

Comment:
'how many users are on specific' - are you sure?

Reply:
fixed

Reply:
fixed